neuronpp/utils/record.py

```diff
@@ -143,17 +151,66 @@ def _plot_animate(self, steps=10000, y_lim=None, position=None):
 
                 ax.set_xlim(t.min(), t.max())
                 if y_lim is None:
-                    ax.set_ylim(r.min()-(np.abs(r.min()*0.05)), r.max()+(np.abs(r.max()*0.05)))
+                    y_limits = (r.min() - (np.abs(r.min() * 0.05)), r.max() + (np.abs(r.max() * 0.05)))
+                    ax.set_ylim(y_limits)
 
                 # update data
                 line.set_data(t, r)
-
+                if show_true_predicted:
+                    # info draw triangles for true and predicted classes
```

Comment:
This whole part should be moved to the separated method. The method `_plot_animate()` is already too big.

Reply:
THe owner is my god